testnet/stacks-node/src/config.rs

```diff
@@ -1348,7 +1355,7 @@ impl MinerConfig {
     }
 }
 
-#[derive(Clone, Default, Deserialize)]
+#[derive(Clone, Default, Deserialize, Debug)]
```

Comment:
Why do all of these structs here need `Debug`?

Reply:
I did that so we can print the config files to screen. I put a part in the PR description about this.

However I'll revert.